### DISCUSSION TITLE
How we can make custom function to use on Tera?

### DISCUSSION THREAD

**Discussion by marciano0695** [loco-rs/loco#1050] [Q&A] [answered]
Hello loco's developers!

I want to make custom function to use on Tera engine, but don't know you to make it.

I already see Tera Docs
[https://keats.github.io/tera/docs/#functions](https://keats.github.io/tera/docs/#functions)

Thank you!

**Comment by kaplanelad** [comment, marked as answer]
Hi,

When creating a new project using `loco new`, navigate to the `view_engine.rs` file located in the `src/initializers` folder.

Within the mutable tera variable, you can append custom filters and functions to extend its functionality.
